content/en/docs/ibm/create-cluster.md

```diff
@@ -132,6 +158,124 @@ Wait until the cluster is deployed and configured. It can take a while for the c
+
+```
+So, if the above list contains the VPC that can be used to deploy your cluster note it's ID. Otherwise, proceed with creating a new VPC as follows.
+
+```shell
+$ ibmcloud is vpc-create my-vpc
+Creating vpc my-vpc in resource group kubeflow under account IBM as user prashsh1@in.ibm.com...
+                                                  
+ID                                             r006-hidden-68cc-4d40-xxxx-4319fa3fxxxx   
+Name                                           my-vpc   
+...  
+```
+
+From the above output save the ID in a variable `VPC_ID` as follows, so that we can use it later.
+
+`$ export VPC_ID=r006-hidden-68cc-4d40-xxxx-4319fa3fxxxx`
+
+* Step 4. Create a subnet.
+  (Note: If you want to use existing subnet, proceed to `List all the existing subnets:` )
+List address prefixes and note the CIDR block corresponding to a Zone, e.g. in below example, for Zone: us-south-3 CIDR block is :
+
+"10.240.128.0/18"
+`$ ibmcloud is vpc-address-prefixes $VPC_ID`
+
+```shell
+Listing address prefixes of vpc r006-hidden-68cc-4d40-xxxx-4319fa3fxxxx under account IBM as user new@user-email.com...
+ID                                          Name                                CIDR block        Zone         Has subnets   Is default   Created   
+r006-xxxxxxxx-4002-46d2-8a4f-f69e7ba3xxxx   rising-rectified-much-brew          10.240.0.0/18     us-south-1   false         true         2021-03-05T14:58:39+05:30   
+r006-xxxxxxxx-dca9-4321-bb6c-960c4424xxxx   retrial-reversal-pelican-cavalier   10.240.64.0/18    us-south-2   false         true         2021-03-05T14:58:39+05:30   
+r006-xxxxxxxx-7352-4a46-bfb1-fcbac6cbxxxx   subfloor-certainly-herbal-ajar      10.240.128.0/18   us-south-3   false         true         2021-03-05T14:58:39+05:30  
+```
+
+Now create a subnet as follows.
+
+`$ ibmcloud is subnet-create my-subnet $VPC_ID us-south-3 --ipv4-cidr-block "10.240.128.0/18"`
+
+```shell
+Creating subnet my-subnet in resource group kubeflow under account IBM as user new@user-email.com...
+                       
+ID                  0737-27299d09-1d95-4a9d-a491-a6949axxxxxx   
+Name                my-subnet
+```
+
+List all the existing subnets:
+
+```$ ibmcloud is subnets
+Listing subnets for generation 2 compute in all resource groups and region us-south under account IBM as user prashsh1@in.ibm.com...
+ID                                          Name                      Status      Subnet CIDR       Addresses     ACL                                                    Public Gateway                             VPC                 Zone         Resource group   
+0737-27299d09-1d95-4a9d-a491-a6949axxxxxx   my-subnet                 available   10.240.128.0/18   16373/16384   husker-sloping-bee-resize                              my-gateway                                 my-vpc              us-south-3   kubeflow   
+```
+
+Record the subnet id as,
+`$ export SUBNET_ID=0737-27299d09-1d95-4a9d-a491-a6949axxxxxx`
```

Comment:
Please don't use `$` in the beginning of each command line (like in the previous version of this guide)